README.md

```diff
@@ -2,22 +2,33 @@
+![image](https://raw.githubusercontent.com/Sjoerd1993/Graphs/main/data/screenshots/sin_cos.png)
 
 
-Graphs is a simple but powerful data manipulation tool developed with the GNOME environment in mind. It opens and plots two-column data files which then easily can be manipulated and transformed. Alternatively, data can be generated from an equation. Both imported data and generated data are treated the same, and can be saved as .txt file for further use.
+Graphs is a simple, yet powerful tool that allows you to plot and manipulate your data with ease. You can import data from two-column files or generate it via an equation. All data can be manipulated using a variety of operations.
+Graphs lets you customize the style of plotted data. You can add and edit stylesheets in detail.
+Graphs is an excellent fit both plotting and data manipulation. The plots created with Graphs can be saved in a variety of formats suitable for sharing and presenting to a wide audience, such as in a scientific publication or presentations. It is also possible to save the plots as vector images, which can be easily edited in programs like Inkscape for further customization and refinement. Graphs is written with the GNOME environment in mind, but should be suitable for any other desktop environment as well.
 
-Graphs can both be used as plotting tool or as a data manipulation tool. There's an extensive number of options to change the plot layouts including, but not limited to the axis labels the used scaling on either axis, the length and width of the ticks used, the thickness of the lines, the linestyle, the fonts and more. The plots can be saved in multiple formats, including vector images for further work in e.g. Inkscape. The data manipulation tools
-include options like:
-  - Selecting and cutting part of the data
+The operations include:
+  - Shifting data
+  - Normalizing Data
+  - Smoothening data
+  - Centering Data
+  - Cutting Data
+  - Combining Data
   - Translating data
-  - Center data
-  - Smoothening of data
-  - Take the derivative or indefinite integral of your data
+  - Derivative and indefinite integral
   - Fourier Transformations
   - Custom transformations using anything that is compatible with the numpy module
  
 For feedback or general issues, please file an issue [at the Github issue tracker](https://github.com/SjoerdB93/Graphs/issues).
 
+## How to build
+This project is developed in [GNOME Builder](https://developer.gnome.org/documentation/introduction/builder.html). After cloning and opening the project, you can press run to verify you have all correct dependencies installed.
+You might need to install meson, if it is not already availabe on your system.
+When the project successfully ran, you can create a flatpak-bundle on the buildchain menu, which you then can install on your system.
+
+If you want to try the latest development, we urge you to try the flathub beta branch instead of building yourself.
```

Comment:
It's also possible to completely build without Flatpak, even though I would kinda advise against. But either way we can add that as possible for people that really want to or have to avoid Flatpak for whatever reason.

From my V
Requirements to build: meson, blueprint-compiler, gettext
Requirements to run: blueprint-compiler, python, matplotlib, python3-matplotlib-gtk4, scipy, numpy, 

These can all be found in Fedora's repo's (so the Python modules can be installed without Pip even which is probably best), same is probably true in most popular distro's.

The commands to then install graphs from source are then:
```
git clone https://github.com/Sjoerd1993/Graphs.git
cd Graphs
meson setup build
ninja -C build
ninja -C build/ install
mkdir -p ~/.local/share/flatpak/exports/share/graphs/graphs && cp data/config.json $_
mkdir -p ~/.config/matplotlib/stylelib && cp data/styles/adwaita.mplstyle $_
mkdir -p ~/.config/matplotlib/stylelib && cp data/styles/adwaita-dark.mplstyle $_
cp data/styles/adwaita-dark.mplstyle ~/.config/matplotlib/stylelib
```

I just tried this in a Fedora 38 virtual environment, and this all works. Uninstall seems to work using `sudo ninja -C build/ uninstall` instead.

Again, this should be accompanied by a clear warning that this is not the supported way to do this. But I think it can be useful to have the instructions up anyway.

Reply:
Now I think of it, we may want to copy all the styles to the Matplotlib install, not just the adwaita ones. So instead of the two separate commands we can do a recursive copy on the folder:


`mkdir -p ~/.config/matplotlib/stylelib && cp -r data/styles/ $_`

Reply:
Also maybe one line that it can be uninstalled again using `sudo ninja -C build/ uninstall` would be good. Otherwise I think this is ready for merging

Reply:
Regarding custom install, I think there is no need to copy the styles and config, as they are copied on runtime anyway?

Reply:
Thanks you are correct! This used to be necessary earlier when I answered in this thread: 
https://github.com/Sjoerd1993/Graphs/discussions/161

However, the get_config_path function has been improved since, and checks the environment variable to see if it's running in Flatpak. I just wanted to create a new PR to implement exactly that solution (it's what I used in another program to fix this) only to find out it was already implemented :)

Also the styling was not copied on runtime at the time of the question in the link. It is now. So I just tested it, and indeed these copy commands are not needed anymore! Which is great as it's definitely cleaner like this!